File: Robot Mesh Studio/C++/Arm Lifting WIth Hold.cpp
//LINK TO SOURCE: https://www.robotmesh.com/studio/5be40c7bc8f17a1f579659df
// VEX V5 C++ Project
#include "vex.h"
using namespace vex;

//#region config_globals
vex::brain      Brain;
vex::motor      motor_arm(vex::PORT1, vex::gearSetting::ratio18_1, false);
vex::controller con(vex::controllerType::primary);
//#endregion config_globals


int main(void) {
    motor_arm.setVelocity(35,pct);
    while(true){
        if (con.ButtonL1.pressing()) {
            //Move arm towards upper limit (assuming arm started at the bottom)
            motor_arm.startRotateTo(110 * 7,deg);
        } else if (con.ButtonL2.pressing()) {
            //Move arm towards lower limit
            motor_arm.startRotateTo(0,deg);
        } else {
            //No driver input, so stop and hold position
            motor_arm.stop(hold);
        }
    }
}

File: Robot Mesh Studio/Offical Vex examples/Python/Arm Lifting with Hold.py
#LINK TO SOURCE: https://www.robotmesh.com/studio/5be40c7ac8f17a1f57964f74
import vex

#region config
brain     = vex.Brain();
motor_arm = vex.Motor(vex.Ports.PORT1, vex.GearSetting.RATIO18_1, False)
con       = vex.Controller(vex.ControllerType.PRIMARY)
#endregion config

# main thread
motor_arm.set_velocity(35, vex.VelocityUnits.PCT)
while True:
  if con.buttonL1.pressing():
    # Move towards the arm's upper limit
    # (Assumes arm started at bottom)
    motor_arm.start_rotate_to((110 * 7), vex.RotationUnits.DEG)
  elif con.buttonL2.pressing():
    # Move towards the arm's lower limit
    # (Assumes arm started at bottom)
    motor_arm.start_rotate_to(0, vex.RotationUnits.DEG)
  else:
    # No driver input, so stop the arm and hold position
    motor_arm.stop(vex.BrakeType.HOLD)